src/Schema/SqliteSchemaManager.php

```diff
@@ -419,7 +419,12 @@ protected function _getPortableTableForeignKeysList($tableForeignKeys)
                 ];
             }
 
-            $list[$name]['local'][]   = $value['from'];
+            $list[$name]['local'][] = $value['from'];
+
+            if ($value['to'] === null) {
```

Comment:
This changes the behavior now. Are `null` values even valid for `to`?

Reply:
This specific change is a backport of 9ddd08a15a291bf1b4f6d7172307f3006859fb77 from `4.0.x`. The table manually declared in `SqliteSchemaManagerTest::testListForeignKeysFromExistingDatabase()` produces the NULL value of `to`. IIRC, it implies that the foreign key references columns with the same names in the referenced tables. Note that the original test expectation doesn't make sense in any case: https://github.com/doctrine/dbal/blob/e814cdc79b804b9ae40f46acbfe191bfc6731b54/tests/Functional/Schema/SqliteSchemaManagerTest.php#L75-L79